### ISSUE TITLE
Elmah 1.2 in nuget Failed to download package correctly. The contents of the package could not be verified.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Benknightdark/elmah#229]
```
What steps will reproduce the problem?

Using the Nuget official package source

Install-Package elmah
Install-Package : Failed to download package correctly. The contents of the 
package could not be verified.
At line:1 char:16
+ Install-Package <<<< elmah
+ CategoryInfo : NotSpecified: (:) [Install-Package], InvalidDataException
+ FullyQualifiedErrorId : 
NuGetCmdletUnhandledException,NuGet.PowerShell.Commands.InstallPackageCommand


What is the expected output? What do you see instead?

To install Elmah 1.2, I got nothing. I opened an issue on Nuget and they said 
it was due to your usage.

http://nuget.codeplex.com/workitem/1086
```

Original issue reported on code.google.com by `android...@gmail.com` on 20 May 2011 at 5:21

**Comment by GoogleCodeExporter** [comment]
```
This is an issue with NuGet and not ELMAH!
We are working with Phil Haack to get this resolved.
```

Original comment by `jamesdriscoll71` on 20 May 2011 at 5:27
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
That's not what Phil Haack said on my linked nuget workitem.
```

Original comment by `android...@gmail.com` on 20 May 2011 at 5:32

**Comment by GoogleCodeExporter** [comment]
```
See http://nugetgallery.codeplex.com/workitem/8 which Phil raised today.
We updated the external URL to point to a *new* package.
NuGet.org let us do that.
NuGet.org then couldn't serve up the package.
Therefore a NuGet problem!!

Anyway, there is now a new version of the package (1.2.0.1) up and ready to go.
```

Original comment by `jamesdriscoll71` on 20 May 2011 at 5:45

**Comment by GoogleCodeExporter** [comment]
```
The NuGet gallery does not currently support replacing an externally hosted 
package, which we learned the hard way. The fact that NuGet still allows you to 
do that makes it all too easy to break things. The version number of the 
"elmah" package had to be bumped up to 1.2.0.1 for the gallery to accept a 
revision on the package even though ELMAH itself still remains at 1.2.

See NuGet issue 8 from ours truly Phil Haack :)…
http://nugetgallery.codeplex.com/workitem/8
```

Original comment by `azizatif` on 20 May 2011 at 5:48

**Comment by GoogleCodeExporter** [comment]
```
Meanwhile, your vote on the NuGet Gallery issue would be appreciated… ;)
http://nugetgallery.codeplex.com/workitem/8

```

Original comment by `azizatif` on 20 May 2011 at 5:54

**Comment by GoogleCodeExporter** [comment]
```
@azizatif if I am understanding the situtation, you released Elmah 1.2 to nuget 
with a url to the nupkg. You then updated the nupkg and did not release a new 
version to nuget. At this point the new nupkg has a hash mismatch with the 
original version, yes?

If what I said is the case Nuget got this absolutely right. You should never be 
able to release multiple versions of the same version number in Nuget for any 
reason, ever.

Releasing 1.2.0.1 is the correct solution.
```

Original comment by `android...@gmail.com` on 23 May 2011 at 1:57

**Comment by GoogleCodeExporter** [comment]
```
I think this is really a debate that should occur on the NuGet Gallery work 
item (http://nugetgallery.codeplex.com/workitem/8) rather than here, as that's 
where the problem lies!!

Regardless of your viewpoint, I used the gallery via its public GUI and 
performed valid actions which did not raise any errors. The gallery *needs* 
fixing so that either a) you cannot update the external URL or b) you can and 
the gallery does whatever it needs to so as not to break packages.

This is *defintely* not an ELMAH problem!! :O)

```

Original comment by `jamesdriscoll71` on 23 May 2011 at 4:31
